dotcom-rendering/src/web/components/Callout/MessageUs.tsx

```diff
@@ -78,22 +67,30 @@ export const MessageUs = ({ format }: CalloutContactProps) => {
 			>
 				Message us on Whatsapp
 			</LinkButton>
-
+			<LinkButton
+				data-ignore="global-link-styling"
+				cssOverrides={calloutPrimaryButton}
+				type="submit"
+				priority="primary"
+				href={OPEN_TELEGRAM_URL}
+				target="_blank"
```

Comment:
Will add this in! For future reference, it looks like this may not even be necessary? The page says...

Note: Setting `target="_blank"` on <a> elements now implicitly provides the same rel behavior as setting `rel="noopener"` which does not set window.opener. See [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility) for support status.